src/NewTwitchApi/Resources/GamesApi.php

```diff
@@ -27,20 +27,16 @@ public function getGames(array $ids = [], array $names = []): ResponseInterface
     }
 
     /**
-     * @param int $first The number of items to return, default is 20
-     * @param null|string $cursor The pagination cursor from a previous request
-     * @param bool $after Return results before (false) or after (true) the cursor
-     * @return ResponseInterface
      * @throws GuzzleException
      * @link https://dev.twitch.tv/docs/api/reference/#get-top-games
      */
-    public function getGamesTop($first = 20, $cursor = null, $after = true): ResponseInterface
+    public function getGamesTop(int $first = 20, string $cursor = null, bool $after = true): ResponseInterface
```

Comment:
According to the Twitch documention in your `@link` URL, `after` is a string, not a boolean.